spec/rubocop/cop/style/numeric_literal_prefix_spec.rb

```diff
@@ -0,0 +1,123 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Style::NumericLiteralPrefix, :config do
+  subject(:cop) { described_class.new(config) }
+
+  context 'octal literals' do
+    context 'when config is zero_with_o' do
```

Comment:
Ohh, I assumed the default would be present in `cop_config['EnforcedOctalStyle']`. Will add it.